singa_auto/model/dev.py

```diff
@@ -240,20 +245,24 @@ def make_predictions(queries: List[Any], task: str,
         predictions_at_predictor.append(prediction)
 
     # Predictor ensembles predictions
-    ensemble_method = get_ensemble_method(task)
-    print(f'Ensemble method: {ensemble_method}')
-    out_predictions = []
-    for prediction in predictions_at_predictor:
-        prediction = prediction.prediction
-        _assert_jsonable(
-            prediction,
-            Exception('Each `prediction` should be JSON serializable'))
-        out_prediction = ensemble_method([prediction])
-        out_predictions.append(out_prediction)
-
-    print('Predictions: {}'.format(out_predictions))
+    if task == 'question_answering':
```

Comment:
there are different question_answering tasks.
this check will never pass as there is no task named "question_answering"

Reply:
conflicts were caused due to previous commit #88: https://github.com/nusdbsystem/singa-auto/commit/13ce73c466835fc9e542645ee09561882a293238#diff-0986337b1fef5d87f529f1e45b9405f1

I have discussed it with Naili, and now the conflict solved